doc/docs/master/concepts/pipeline-concepts/datum/relationship-between-datums.md

```diff
@@ -38,127 +29,84 @@ and data generation workflows.
+    - Each file is put in specific directories. This directory structure has been thought to facilitate the aggregation of the content in the following pipeline. Think about your directory structure so that the next glob pattern will aggregate your data as needed.
 
-- `datum 1` has one file and results in files `1` and `2`.
-- `datum 2` has one file and results in files `2` and `3`.
-- `datum 3` has one file and results in files `1` and `3`.
 
-![One to many](../../../assets/images/d_datum_processing_one_to_many.svg)
+![Map Reduce](../../../images/parallel_data_processing.png)
 
-Pachyderm processes all these datums independently, and in the end, it needs
-to create a commit by combining the results of processing these datums.
-A commit is a filesystem that has specific constraints, such as duplicate
-files with the same file path. Pachyderm merges results from
-different output datums with the same file path into single files. For
-example, `datum 1` produces `pfs/out/1` and `datum 3` produces `pfs/out/1`.
-Pachyderm merges these two files by appending them one to another
-without any particular order. Therefore, the file `1` in the final
-commit has parts from `datum1` and `datum2`.
 
-If you decide to create a new commit and overwrite the file in `datum 2`,
-Pachyderm detects three datums. Because `datum 1` and `datum 3` are
+Let's now create a new commit and overwrite a file in `datum 2`,
+Pachyderm detects three datums. However, because `datum 1` and `datum 3` are
 unchanged, it skips processing these datums. Then, Pachyderm detects
 that something has changed in `datum 2`. Pachyderm is unaware of any
-details of the change. Therefore, it processes the whole `datum 2`
-and outputs the files `1`, `3`, and `4`. Then, Pachyderm merges
-these datums to create the following final result:
+details of the change; therefore, it processes the whole `datum 2`
+and outputs 3 files. Then, the following pipeline aggregates
+these data to create the following final result:
 
-![One to many](../../../assets/images/d_datum_processing_one_to_many_overwrite.svg)
+![Map Reduce](../../../images/parallel_data_processing_following_commit.png)
 
+## Incrementality 
+In Pachyderm, glob patterns are applied to the entire input, 
+however, unchanged datums are never re-processed. 
+For example, if you have multiple datums, 
+and only one datum was modified; Pachyderm processes that datum only
+and `skips` processing other datums. 
+This incremental behavior ensures efficient resource utilization.
 
-In the diagram above, Pachyderm appends the file `1` from the `datum 2`
-to the file `1` in the final commit, deletes the file `2` from `datum 2`,
-overwrites the old part from `datum 2` in file `3`  with a new version,
-and creates a new output file `4`.
-
-Similarly, if you have multiple files in your input datum, Pachyderm might
-write them into multiple files in output datums that are later merged into
-files with the same file path.
+## Overwriting files
```

Comment:
This seems like it belongs in a different doc, probably with other PFS file operations.

Reply:
Noted - This belongs to the file.md page. I still want to mention the default 'overwrite' behavior but then point to the full info on the appropriate page.